python/paddle/fluid/imperative/base.py

```diff
@@ -25,18 +25,29 @@ def enabled():
 
 
 @contextlib.contextmanager
-def guard():
+def guard(device=0):
```

Comment:
remove this API change. infer from input tensor

Reply:
Without this API, we should have a way to make parameters be put in GPU

Reply:
And a way to make op without input keep in GPU, e.g.. ```fill_constant```